airflow/operators/bash_operator.py

```diff
@@ -89,6 +89,7 @@ def execute(self, context):
         bash_command = ('export {}={}; '.format(AIRFLOW_HOME_VAR, airflow_home_value) +
                         'export {}={}; '.format(PYTHONPATH_VAR, pythonpath_value) +
                         self.bash_command)
+        self.lineage_data = bash_command
```

Comment:
Can't we also do this using a decorator, like we do with the `{pre,post}_execute`.

Reply:
How to do that? It is an arbitrary assignment, so how do I know which variable to catch?